plugin.video.digistorageplayer/addon.py

```diff
@@ -0,0 +1,104 @@
+import sys
+import urllib
+import urlparse
+import xbmcgui
+import xbmcplugin
+import xbmcaddon
+import requests
+from xbmc import log as xbmc_log
+from urlparse import parse_qsl
+
+requests.packages.urllib3.disable_warnings()
+
+base_url = sys.argv[0]
+addon_handle = int(sys.argv[1])
+args = urlparse.parse_qs(sys.argv[2][1:])
+
+xbmcplugin.setContent(addon_handle, 'movies')
```

Comment:
Are you sure that "movies" is a good decision for content type? I would suggest to use "files" i think.